src/plugins/data/public/ui/saved_query_flyouts/open_saved_query_flyout.tsx

```diff
@@ -111,6 +122,7 @@ export function OpenSavedQueryFlyout({
     fetchAllSavedQueriesForSelectedTab();
     setSearchQuery(EuiSearchBar.Query.MATCH_ALL);
     updatePageIndex(0);
+    setSelectedQuery(undefined);
```

Comment:
curious, why do you have to do this?

Reply:
nevermind, assuming you want to unset when reloading...

Reply:
Yes when the queries are reloaded or tab is changed